lisp/ess-sas-a.el

```diff
@@ -809,14 +809,14 @@ optional argument is non-nil, then set-buffer rather than switch."
   "And now for something completely different."
   (interactive)
   ;;(ess-sas-file-path)
-  (setq ess-customize-alist SAS-customize-alist)
+  (setq ess-local-customize-alist SAS-customize-alist)
```

Comment:
This is redundant and can be removed I think, because `ess-local-customize-alist` is itself part of the cust-alists. Then `ess-setq-vars-local` can be passed `SAS-customize-alist` directly as in the R runner.